sources/packages/api/src/services/student/student.service.ts

```diff
@@ -32,6 +40,8 @@ export class StudentService extends RecordDataModelService<Student> {
     const student = await this.repo
       .createQueryBuilder("student")
       .leftJoinAndSelect("student.user", "user")
+      .innerJoin("student.sinValidation", "sinValidation")
```

Comment:
is it `leftJoin`?

Reply:
This relation will always exist @ann-aot.
The student record must be created with it.

Reply:
but DB column is nullable, 
![image](https://user-images.githubusercontent.com/77353155/157532553-67802aae-a4c0-4e03-b013-a1ea07c02383.png)

Reply:
I added a comment about it also. It must be NOT NULL 😉